lib/postgap/Cisreg.py

```diff
@@ -110,82 +109,20 @@ def _snp_rest(self, snp):
 			Returntype: [ Cisregulatory_Evidence ]
 
 		"""
-		cisreg_with_pvalues = self._snp_pvalues(snp)
-		
-		if cisreg_with_pvalues is None:
+		full_cisreg=self._snp_values(snp)
+		if full_cisreg is None:
+			logging.warning("Got exception in _snp_rest")
```

Comment:
Why put a warning on two lines? Putting them on a single would ensure there is no ambiguity.